library/amazonlinux

```diff
@@ -4,13 +4,21 @@ Maintainers: Amazon Linux Team <amazon-linux@amazon.com> (@aws),
              Eric Warehime (@Eric-Warehime)
 GitRepo: https://github.com/aws/amazon-linux-docker-images.git
 
-Tags: 2.0.20181010, 2, latest
-GitFetch: refs/heads/amzn2
-GitCommit: 1a13f4b999a22b95b0d8bbd10d4190e9fc1b43fd
+Tags: 2.0.20181114, 2, latest
+Architectures: amd64, arm64v8
+GitCommit: 3de85bea51cc97f74263a12825564db727b55996
```

Comment:
So, this is fine, but could potentially lead to `amd64` bits being pushed to a different architecture (typo in `ARCH-GitCommit:` field name, for example), so I'd recommend pointing this global value to some useless commit from `master` that'll explicitly fail to build (and then you can move it up to the global section as well so it doesn't have to be duplicated).

Reply:
OK, I'll just use the latest commit from master, which is missing a Dockerfile.

Reply:
I missed your text about moving it up to the global section... I'll do that next time :)